docs/content/aws.md

```diff
@@ -0,0 +1,391 @@
+# Running El Carro Operator on Amazon cloud (AWS)
+
+This guide helps you run El Carro Operator in [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/).
+
+If you prefer to use GKE (Google Kubernetes Engine) to deploy the El
+Carro Operator, stop here and refer to the [Quickstart Guide](quickstart.md) or
+[Quickstart Guide for Oracle 18c XE](quickstart-18c-xe.md).
+
+If you prefer to use a local cluster on your personal computer instead of AWS EKS, refer to either the [Minikube Guide](minikube.md) or [Kind Guide](kind.md).
+
+## Before you begin
+
+We used a t2.micro EC2 instance to run this test, as of today the [AWS CloudShell doesn’t support Docker containers](https://docs.aws.amazon.com/cloudshell/latest/userguide/vm-specs.html).
+All steps were executed as the ec2-user from within the EC2 instance:
+
+## Install docker, kubectl and eksctl 
+
+* Install Docker - needed to check images in repository or build images locally.
+
+    ```shell script
+    sudo amazon-linux-extras install docker
+    sudo yum update -y
+    sudo service docker start
+    sudo usermod -a -G docker ec2-user
+    reboot
+    ```
+
+* Install [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) to interact with Kubernetes clusters.
+
+    ```shell script
+    curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/kubectl
+    chmod +x ./kubectl
+    mkdir -p $HOME/bin && mv ./kubectl $HOME/bin/kubectl && export PATH=$PATH:$HOME/bin
+    echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
+    kubectl version --short --client
+    Client Version: v1.20.4-eks-6b7464
+    ```
+
+*  Install [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) to interact with the EKS cluster, which automates many individual tasks.
+
+    ```shell script
+    curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+    sudo mv /tmp/eksctl /usr/local/bin
+    eksctl version
+     0.55.0
+    ```
+
+*   Make sure you have access to the latests El Carro source code from Github 
+
+    ```shell script
+    wget https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/releases/download/v0.0.0-alpha/release-artifacts.tar.gz
```

Comment:
Can you please change this link to `https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/releases/latest` or point readers to it? The aforementioned link will always point to the latest El Carro release.

Reply:
Fixed in next commit